### PR TITLE
Centralized Elasticsearch indexer module config file

### PR DESCRIPTION
**Pull request by jayrbolton**
This adds a centralized YAML config for a lot of the options we had spread among different modules inside the es_indexers package. 

- The config file in `spec/elasticsearch_modules.yaml` maps a KBase workspace type to a python module that does the indexing (a `main` function needs to be in each module)
- Each indexer entry also has config data for things like index name, subobject index name, namespaces, etc
- The schema describing the format of the config file is inside the same file under the "indexers_schema" key

* [x] I incremented the version in the `VERSION` file
* [x] I updated `CHANGELOG.md`
* [x] I added test coverage for any changes

### CODE REVIEW

**Comment by jayrbolton** [comment]
@slebras just resolved the conflict w develop

File: tests/integration/test_integration.py [hunk 1]
```diff
@@ -8,14 +8,17 @@
...
 _TEST_EVENT = {
    "wsid": 55825,
    "ver": 15,
    "perm": None,
    "evtype": "NEW_VERSION",
    "objid": 2,
    "time": 101,
-   "user": "jayrbolton"
+   "user": "jayrbolton",
+   "index_runner_ver": ["1.9.10"],
```
Comment (slebras) [inline tests/integration/test_integration.py]: should this be updated to 1.9.11

**Review by slebras — APPROVED** [review]
looks good, just 1 comment

File: tests/integration/test_integration.py (hunk 1 above)
Comment (jayrbolton) [reply tests/integration/test_integration.py]: Well this is really just fake test data. I'm not sure it matters, but if we want it to always match the current version then we should load it from the config. I'll look at it a little

**Pull request merged**